ocaml-lsp-server/src/document.ml

```diff
@@ -281,13 +281,23 @@ module Merlin = struct
     | `Found s | `Builtin s -> Some s
     | _ -> None
 
+  let syntax_doc pipeline pos =
+    let res =
+      let command = Query_protocol.Syntax_document pos in
```

Comment:
This query is available in only recent version of merlin, right? If so, an appropriate constraint should be added to the ocaml-lsp-server package (in dune-project).